src/package.json

```diff
@@ -28,14 +29,17 @@
   },
   "homepage": "https://github.com/LeoPlatform/serverless-leo#readme",
   "dependencies": {
+    "aws-sdk": "^2.485.0",
```

Comment:
I think there might be a way to set this up as a peer dependency, so our node module doesn't actually install the aws-sdk at any particular version. It just makes sure it has access to it.